apps/backoffice-v2/src/common/utils/fetcher/fetcher.ts

```diff
@@ -49,8 +49,6 @@ export const fetcher: IFetcher = async ({
         : message;
     }
 
-    console.error(message);
-
```

Comment:
We might want to at least make the log optional or provide some other indicator for errors.